### PR TITLE
feat: proposal hero

### LINKED ISSUE #1275: Proposal view hero
Part of #1263

### PR DESCRIPTION
**Pull request by 1emu**
Closes #1275 

- Hero banner for active proposal types, white Pill & transparent pill

image
image
image
image
image
image
image
image

- Rejected/Passed/Finished

image
image
image

- Long text

image
image
image

### CODE REVIEW

File: src/components/Common/Pill.tsx [hunk 1]
```diff
@@ -14,13 +14,15 @@ export enum PillColor {
...
   Aqua = 'aqua',
+  White = 'white',
 }
 
 export type Props = {
   children: React.ReactText
   color?: PillColor | `${PillColor}`
   size?: 'sm' | 'md'
   style?: 'shiny' | 'medium' | 'light' | 'outline'
+  transparent?: boolean
```
Comment (andyesp) [inline src/components/Common/Pill.tsx:25]: I'd change this so transparent is also a color `color="transparent"`

File: src/components/FloatingBar/FloatingBar.css [hunk 2]
```diff
@@ -60,3 +60,9 @@
   margin-right: 84px !important;
   position: relative !important;
 }
+
+@media (max-width: 374px) {
```
Comment (andyesp) [inline src/components/FloatingBar/FloatingBar.css:64]: I'd do the opposite here. `display: none` as default and add a min-width media query so it has a mobile-first approach.

File: src/components/Proposal/HeroBanner.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  border: 1px solid #e8e8e8;
+  background: linear-gradient(89deg, #f6f5f8 0%, rgba(246, 245, 248, 0) 100%);
+}
+
+.HeroBanner__Gradient {
+  background: linear-gradient(90deg, rgba(22, 20, 26, 0) 0%, #16141a 100%);
+}
+
+.HeroBanner--purple {
+  background: #a524b3;
```
Comment (andyesp) [inline src/components/Proposal/HeroBanner.css:19]: Can we add this to the theme? Can't we use purple-800?

File: src/components/Proposal/HeroBanner.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import classNames from 'classnames'
+
+import { PillColor } from '../Common/Pill'
+
+import './HeroBanner.css'
+
+export default function HeroBanner(props: { proposalActive: boolean; colorsConfig: PillColor }) {
```
Comment (andyesp) [inline src/components/Proposal/HeroBanner.tsx:9]: `color` instead of `colorsConfig`?

File: src/components/Proposal/HeroBanner.css (hunk 3 above)
Comment (1emu) [reply src/components/Proposal/HeroBanner.css:19]: This is purple-A or B in figma. 

Added --fuchsia-500

image

image

**Review by ncomerci — APPROVED** [review]

**Review by andyesp — APPROVED** [review]

**Pull request merged**